### ISSUE TITLE
Community Communcation Discord/Slack?

### ISSUE THREAD

**Issue by johnmcbride** [elsa-workflows/elsa-core#4722]
Is there a discord/slack community that we can join to get help when trying to build solutions ontop of ELSA?

**Comment by sNakiex** [comment]
Yes there is a community discord: https://discord.gg/hhChk5H472 The links to other community tools (like reddit) are all in the readme!
